setup.sh

```diff
@@ -87,7 +87,7 @@ if [[ "$1" != "" ]]; then
 				extra_args="--seed"
 			fi
 		fi
-		audius-cli launch $extra_args "$1" --force
+		audius-cli launch $extra_args "$1" # do not pass --yes so that we check for machine requirements
```

Comment:
@cheran-senthil oh yes i forgot about the system requirements check. the reason i did this was we ask launch the service which seems redundant because this asks to launch the service again